fineract-provider/src/main/resources/static/api-docs/velocity-doc.md

```diff
@@ -0,0 +1,900 @@
+## API documentation.
+
+### **Authentication**
+Fineract APIs are secured and depending on the security options available in fineract, any api request would require some form of authentication and authorization. Checkout this documentation for more information about available authentication protocols. 
+
+[<u>https://devcredits.velocitycareerlabs.io/fineract-provider/api-docs/apiLive.htm#authentication_overview</u>](https://devcredits.velocitycareerlabs.io/fineract-provider/api-docs/apiLive.htm#authentication_overview)  
+NOTE: Staging instance currently uses Basic Authentication and as such base 64 encoded username and password passed in the Authorization headers of any http request should guarantee access/authorization. 
```

Comment:
Wasn't the integration with `auth0` already done?

Reply:
This documentation doesn't necessarily cover that and no, auth0 is not yet done